apicast/src/threescale_utils.lua

```diff
@@ -164,6 +174,11 @@ function _M.connect_redis(options)
   return red
 end
 
+-- return ownership of this connection to the pool
+function _M.release_redis(red)
+  red:set_keepalive(redis_conf.keepalive, redis_conf.poolsize)
+end
```

Comment:
Nice! :)
Hopefully we can also extract all redis-related stuff to a separate module in a next iteration (not this PR I guess)